install.sh

```diff
@@ -9,77 +17,95 @@ log () {
 then
-    log "Das Paket \"wget\" wurde nicht gefunden, wird aber benötigt. Es wird nun installiert..."
+    echo -e "$YELLOWℹ\"wget\" ist nicht installiert.$NORMAL Die Installation wurde gestartet..."
     sleep 2
     apt-get install wget -y
 fi
 
 # Check for unzip
+clear
+echo -e "$BLUE🔎 Status:$NORMAL Überprüfe, ob unzip vorhanden ist..."
 if ! command -v unzip &> /dev/null
 then
-    log "Das Paket \"unzip\" wurde nicht gefunden, wird aber benötigt. Es wird nun installiert..."
+    echo -e "$YELLOWℹ\"unzip\" ist nicht installiert.$NORMAL Die Installation wurde gestartet..."
     sleep 2
     apt-get install unzip -y
 fi
 
 # Check for curl
+clear
+echo -e "$BLUE🔎 Status:$NORMAL Überprüfe, ob curl vorhanden ist..."
 if ! command -v curl &> /dev/null
 then
-    log "Das Paket \"curl\" wurde nicht gefunden, wird aber benötigt. Es wird nun installiert..."
+    echo -e "$YELLOWℹ\"curl\" ist nicht installiert.$NORMAL Die Installation wurde gestartet..."
     sleep 2
     apt-get install curl -y
 fi
 
 # Check for node
+clear
+echo -e "$BLUE🔎 Status:$NORMAL Überprüfe, ob node vorhanden ist..."
 if ! command -v node &> /dev/null
 then
-    log "Das Paket \"nodejs\" wurde nicht gefunden, wird aber benötigt. Es wird nun installiert..."
+    echo -e "$YELLOWℹ\"node\" ist nicht installiert.$NORMAL Die Installation wurde gestartet..."
     sleep 2
     curl -sSL https://deb.nodesource.com/setup_16.x | bash
     apt-get install nodejs -y
 fi
 
-log "Alle notwendigen Pakete sind installiert. Starte installation von MySpeed..."
-sleep 2
+clear
+echo -e "$GREEN✓ Vorbereitung abgeschlossen:$NORMAL Die Installation von MySpeed wird jetzt gestartet..."
+sleep 3
 
+clear
 if [ ! -d $INSTALLATION_PATH ]
 then
-    log "MySpeed wird unter $INSTALLATION_PATH installiert. Der Ordner wird nun erstellt."
+    echo -e "$BLUEℹ Info: $NORMAL MySpeed wird unter dem Verzeichnis $INSTALLATION_PATH installiert. Der Ordner wird nun erstellt."
     sleep 2
     mkdir $INSTALLATION_PATH
 fi
 
 cd $INSTALLATION_PATH
 
-log "Lade notwendige Daten herunter..."
+echo -e "$BLUEℹ Info: $NORMAL Die aktuelle MySpeed-Instanz wird heruntergeladen. Einen Moment..."
 sleep 2
 wget "$RELEASE_URL"
 
-log "Entpacke notwendige Daten..."
+echo -e "$BLUEℹ Info: $NORMAL Download abgeschlossen. Entpacken läuft..."
 sleep 2
 unzip MySpeed*.zip
 rm MySpeed-*.zip
 
-log "Lade die notwendigen Abhängigkeiten herunter..."
+echo -e "$BLUEℹ Info: $NORMAL Die notwendigen Abhängigkeiten werden jetzt installiert..."
 sleep 2
 npm install
 
 clear
-log "Erfolg! MySpeed wurde unter $INSTALLATION_PATH installiert."
\ No newline at end of file
+echo -e "$GREEN-$NORMAL-$GREEN-$NORMAL$GREEN-$NORMAL$GREEN-$NORMAL$GREEN-$NORMAL$GREEN-$NORMAL$GREEN-$NORMAL$GREEN-$NORMAL$GREEN-$NORMAL" #multicolor
+echo -e "$GREEN✓ Installation abgeschlossen: $NORMAL MySpeed wurde unter $INSTALLATION_PATH installiert."
+echo -e "Die Weboberfläche findest du im Browser unter $BLUEhttp://localhost:5216$NORMAL."
```

Comment:
Hier müsste man das Leerzeichen verschieben, weil sonst Bash das "http" noch als Variable erkennt
```suggestion
echo -e "Die Weboberfläche findest du im Browser unter$BLUE http://localhost:5216$NORMAL."
```